ode/unit_test/Test_ODE_RK.hpp

```diff
@@ -475,7 +568,13 @@ int test_RK_conv_rate() {
   return 1;
 }
 
+int test_RK_adaptivity() {
+  Test::test_adaptivity<TestExecSpace>();
+  return 1;
```

Comment:
return 0, for success?

Reply:
No, gtest does not check for success by capturing the return value of our tests.
I actually supspect that gtest assumes our test function to be void?